pixi.toml

```diff
@@ -0,0 +1,49 @@
+[project]
```

Comment:
This file should have a leading comment explaining:

* what pixi is
* how to install pixi
* how to run pixi commands